src/core/PageNavigation.tsx

```diff
@@ -44,21 +45,20 @@ const PageNavigation: React.FC<propTypes> = ({
           currentView && currentView.toLowerCase() !== "pathways",
       })}
     >
-      {pageOptions.map((page) => (
-        <li key={page}>
+      {pageOptions.map((pageOption: any) => (
```

Comment:
```suggestion
      {pageOptions.map((pageOption: keyof Navigation) => (
```

Or we could create a new type in `types/navigation.ts`:
```
export type NavigationSection = keyof Navigation;
```